src/main/java/com/xceptance/xlt/mastercontroller/Main.java

```diff
@@ -163,7 +163,7 @@ private static File getOverridePropertieFile(final CommandLine commandLine)
      * @param config
      *            master controller configuration
      */
-    private static void setupHttpsProxy(final MasterControllerConfiguration config)
+    private void setupHttpsProxy(final MasterControllerConfiguration config)
```

Comment:
Shouldn't this be `protected` to allow overrides?

Reply:
Done.